### PR TITLE
Initial user flow views

### PR DESCRIPTION
**Pull request by pbb34**
Updated views

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,9 @@ To run Icebreaker complete the following steps:
 
 1. Get the most recent version of the code
 	- Run `git pull origin <branch>`
-2. Install NPM components
-	- Run `npm install`
+2. Install NPM components and create a package.json file inside Incebreaker project directory
```
Comment (catvec) [inline README.md:28]: Remove this change. `npm install` is the correct command.

File: src/www/assigned-topics.html [hunk 2]
```diff
@@ -16,89 +16,66 @@
...
-        <h3>Select topics 5 </h3>
+        <a href="index.html" data-icon="delete">Cancel</a>
+        <h3>Assigned Topics </h3>
     </div>
 
 
     <ul data-role="listview" data-split-icon="info" data-split-theme="a" data-inset="true">
         <li><a href="#">
-            <img src="/img/sports.png">
+            <img src="img/sports.png">
```
Comment (catvec) [inline src/www/assigned-topics.html:26]: Make all your urls absolute. Meaning they start with a slash.  

Ex: `/img/sports.png`.

**Review by catvec — CHANGES_REQUESTED** [review]
Good work so far. Only a minor issue with some image URLs. Other than that everything is ay-ok.

**Comment by pbb34** [comment]
I’ll fix it boss

**Comment by catvec** [comment]
:ok_hand:

**Comment by pbb34** [comment]
Making URL's absolute was throwing errors such as "Unable to load the content" and images didn't load up. So keeping is same.

**Pull request closed without merging**